xdcs-frontend/src/app/element/task-definition-preview/task-definition-preview.component.html

```diff
@@ -12,62 +12,47 @@
       <h2>{{definition.name}}</h2>
     </div>
   </div>
-  <div class="row">
-    <div class="col-6">
-      <table class="table table-borderless">
-        <tbody>
-        <tr>
-          <th scope="row">ID</th>
-          <td>{{definition.id}}</td>
-        </tr>
-        <tr>
-          <th scope="row">Type</th>
-          <td>{{definition.config.type}}</td>
-        </tr>
-        </tbody>
-      </table>
-    </div>
 
-    <div class="col-6">
-      <h4 class="mb-3">Deployments</h4>
-      <table *ngIf="deploymentDescriptors && deploymentDescriptors.items.length > 0; else noDeployments"
-             class="table w-100">
-        <colgroup>
-          <col/>
-          <col/>
-          <col style="width: 60%;"/>
-        </colgroup>
-        <thead>
-        <tr>
-          <th scope="col">ID</th>
-          <th scope="col">Deployed</th>
-          <th scope="col">Description</th>
-        </tr>
-        </thead>
+  <app-task-definition-details [definition]="definition"></app-task-definition-details>
+  <br>
```

Comment:
po co break?

Reply:
zeby byla przerwa, bo imo było za blisko

Reply:
to margines zamiast breaka